tests/containers/container_engine.pm

```diff
@@ -94,7 +94,7 @@ sub basic_container_tests {
 
     ## Test `--init`, i.e. the container process won't be PID 1 (to avoid zombie processes)
     # expected output: the `ps` command is not running as PID 1. The $runtime-init process shall be 1 instead.
-    validate_script_output("$runtime run --rm --init $image ps --no-headers -xo 'pid args'", sub { $_ =~ m/\s*1 .*${runtime}-init .*/ });
+    validate_script_output("$runtime run --rm --init $image ps --no-headers -xo 'pid args'", sub { $_ =~ m/\s*1 .*(${runtime}-|\/)init .*/ });
```

Comment:
I would consider the following more obvious:

```suggestion
    validate_script_output("$runtime run --rm --init $image ps --no-headers -xo 'pid args'", sub { $_ =~ m/\s*1 .*(${runtime}-)?init .*/ });
```

One might also consider to check for anything with `init`, because the runtime is optional in the regex anyway.

Reply:
No, please just this is wanted here to just expect before `init` either `<name of runtime>-` or the '/' slash and no other.

Reply:
OK, I understand. By putting the slash in the place of the runtime you ensure that the runtime is exactly not optional. That's a great idea - I haven't thought this far.

Reply:
Yes, in fact, mainly init process are only:
`/run/podman-init`, `/sbin/docker-init`, `/dev/init`,

but excluding the folder name it is generic enough.